packages/driver/src/cypress/error_utils.ts

```diff
@@ -197,20 +195,20 @@ const makeErrFromObj = (obj) => {
   return err2
 }
 
-const throwErr = (err, options = {}) => {
+const throwErr = (err, options: any = {}) => {
```

Comment:
```suggestion
const throwErr = (err, options: any = {}): never => {
```

Reply:
As `never` means a function never reaches its end, I had to create `makeErrFromErr` function.